theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/colors.ts

```diff
@@ -0,0 +1,5 @@
+export const COLOR_POPOVER_BACK = 'black'
+
+export const COLOR_BASE = '#292929'
+
+export const COLOR_FOCUS_OUTLINE = '#f78104'
```

Comment:
Thinking out loud: Separating colors out like this feels a bit strange to me. I'm going to be keeping my eye on this kind of pattern and whether there's something interesting we'd want to move towards with custom theming / custom branding for users.

Reply:
Echoing @colelawrence 's suggestion to use css custom properties, there's new-ish rgb functional notation `--clr-base: 29 29 29;` which can be used like so: `background-color: rgb(var(--clr-base) / 50%)` for dynamic modification.  So you can do alpha math in the previous example but also more complex math on the colors when combined with `calc`. 

Then you only need to define a few theme colors and can generate high contrast or complementary colors on demand from that small starting set (which could be user customizable).

Reply:
These are really interesting and useful points and techiques @colelawrence and @thegoleffect. I am going to take note of them. I think it is out of scope for this PR but I want to come back to this.